src/ontology/mondo-edit.obo

```diff
@@ -170630,17 +170630,19 @@ property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/
 
 [Term]
 id: MONDO:0008654
-name: nystagmus 4, congenital, autosomal dominant
+name: spinocerebellar ataxia 27A
 subset: inferred_rare
 subset: rare
 synonym: "NYS4" EXACT ABBREVIATION [MONDO:Lexical, OMIM:193003]
 synonym: "NYSTAGMUS 4, congenital, autosomal dominant" RELATED [MONDO:Lexical, OMIM:193003]
```

Comment:
@nicolevasilevsky , this should not be a related synonym, it should be exact (based on the same other synonym.

Reply:
fixed

there are a lot of cases where OMIM synonyms are related but should be exact. We have an open ticket for this:
https://github.com/monarch-initiative/mondo/issues/2629

I'd be happy to work on this if you think it is a priority.